node/src/components/consensus/protocols/highway.rs

```diff
@@ -840,11 +841,9 @@ where
                     (ValidatorIndex, &IndexObservation),
                     &IndexObservation,
                 )| {
-                    match (our_obs, their_obs) {
+                    match (*our_obs, *their_obs) {
                         (our_obs, their_obs) if our_obs == their_obs => vec![],
 
-                        (IndexObservation::None, IndexObservation::None) => vec![],
```

Comment:
So I'm seeing here `IndexObservation::None` -> `IndexObservation::Count(0)`.  Is that right?

Also, to make sure I'm getting this right: we had this distinction and `Correct` started with 1? This is to @goral09 's point earlier.

Reply:
`Correct(n)` corresponds to `Count(n + 1)`: If the last unit has sequence number `n`, that means we have `n + 1` units. (Sequence numbers start at 0.)